File: .NET/Microsoft.Recognizers.Text.DateTime/Extractors/IDateExtractor.cs
﻿// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using System.Text.RegularExpressions;

namespace Microsoft.Recognizers.Text.DateTime
{
    public interface IDateExtractor : IDateTimeExtractor
    {
        int GetYearFromText(Match match);
    }
}

File: Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/extractors/IDateExtractor.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.recognizers.text.datetime.extractors;

import com.microsoft.recognizers.text.utilities.Match;

public interface IDateExtractor extends IDateTimeExtractor {
    int getYearFromText(Match match);
}
